tools/run_tests/run_xds_tests.py

```diff
@@ -2541,62 +2541,96 @@ def create_global_forwarding_rule(gcp,
 
 
 def get_health_check(gcp, health_check_name):
-    result = gcp.compute.healthChecks().get(
-        project=gcp.project, healthCheck=health_check_name).execute()
-    gcp.health_check = GcpResource(health_check_name, result['selfLink'])
+    try:
+        result = gcp.compute.healthChecks().get(
+            project=gcp.project, healthCheck=health_check_name).execute()
+        gcp.health_check = GcpResource(health_check_name, result['selfLink'])
+    except Exception as e:
```

Comment:
Should we consider only catching errors with type `googleapiclient.errors.HttpError`?

Reply:
I think we still want to try to clean up regardless of the error type.